libraries/standard/mqtt/cbmc/include/mqtt_config.h

```diff
@@ -0,0 +1,44 @@
+#ifndef MQTT_CONFIG_H_
+#define MQTT_CONFIG_H_
+
+/**************************************************/
+/******* DO NOT CHANGE the following order ********/
+/**************************************************/
+
+/* Logging related header files are required to be included in the following order:
+ * 1. Include the header file "logging_levels.h".
+ * 2. Define LIBRARY_LOG_NAME and  LIBRARY_LOG_LEVEL.
+ * 3. Include the header file "logging_stack.h".
+ */
```

Comment:
I feel like I've seen this before - is this a copy?  Can we include the original?  Also - this text is rather unfriendly.  It would be better to explain why a given order is necessary.

Reply:
The proofs need to have their own mqtt_config.h to disable logging.